src/library/layouts/library/ui/SideBar.tsx

```diff
@@ -66,76 +70,92 @@ export function SideBar() {
-              Cookies
-            </Link>
+        {!session && (
+          <div className="grow space-y-spacing-looser overflow-scroll">
+            <Paper variant="elevated">
+              <span className="block font-bold">
+                Create your first playlist
+              </span>
+
+              <span className=" mt-spacing-tighter-2 block text-sm">
+                It’s easy, we’ll help you
+              </span>
+
+              <Button className="mt-spacing-looser" variant="inverted-light">
+                Create playlist
+              </Button>
+            </Paper>
+
+            <Paper variant="elevated">
+              <span className="block font-bold">
+                Let’s find some podcasts to follow
+              </span>
+
+              <span className="mt-spacing-tighter-2 block text-sm">
+                We’ll keep you updated on new episodes
+              </span>
+
+              <Link
+                className={buttonVariants({
+                  className: "mt-spacing-looser",
+                  variant: "inverted-light",
+                  size: "medium",
+                })}
+                href="#"
+              >
+                Browse podcasts
+              </Link>
+            </Paper>
           </div>
-
-          <Button variant="outlined">
-            <HiOutlineGlobeAlt /> English
-          </Button>
-        </footer>
+        )}
+
+        {session && (
+          <div className="grow space-y-spacing-tighter-4 overflow-scroll">
+            <Suspense fallback={<div>Loading user playlists...</div>}>
+              {/* <UserPlaylistsCompact /> */}
+              {/* <UserPlaylistsList /> */}
+              <UserPlaylistsGrid />
+            </Suspense>
+          </div>
+        )}
+
+        {!session && (
+          <footer className="p-spacing-looser">
+            <div className="mb-spacing-looser flex flex-wrap gap-spacing-tighter text-xs text-color-subdued">
+              <Link href="#" underline="hover" target="_blank">
+                Legal
+              </Link>
+
+              <Link href="#" underline="hover" target="_blank">
+                Safety & Privacy Center
+              </Link>
+
+              <Link href="#" underline="hover" target="_blank">
+                Privacy Policy
+              </Link>
+
+              <Link href="#" underline="hover" target="_blank">
+                Cookies settings
+              </Link>
+
+              <Link href="#" underline="hover" target="_blank">
+                About ads
+              </Link>
+
+              <Link href="#" underline="hover" target="_blank">
+                Accessibility
+              </Link>
+
+              <Link href="#" underline="hover" target="_blank">
+                Cookies
+              </Link>
+            </div>
+
+            <Button variant="outlined">
+              <HiOutlineGlobeAlt /> English
+            </Button>
+          </footer>
```

Comment:
Same for these, could we group them in a component now?

The more we are at upper layers (shared and infra --> entities --> features --> widgets --> pages --> layout --> app), the more we want the code to be as thin as possible (more and more abstraction) and the more readable possible.